community/kernel/src/test/java/org/neo4j/unsafe/impl/batchimport/input/csv/CsvInputBatchImportIT.java

```diff
@@ -452,6 +475,34 @@ private void buildUpExpectedData(
         {
             expectedNodes.put( (String) node.id(), node );
             expectedNodeNames.put( nameOf( node ), node.labels() );
+
+            // Build default verifiers for all the properties that compares the property value using equals
+            assert node.hasIntPropertyKeyIds == false;
+            Map<String,Consumer<Object>> propertyVerifiers = new TreeMap<>();
+            for ( int i = 0; i < node.propertyCount(); i++ )
+            {
+                final Object expectedValue = node.propertyValue( i );
+                Consumer verify = actualValue ->
+                {
+                    assertEquals( expectedValue, actualValue );
+                };
+                propertyVerifiers.put( (String) node.propertyKey( i ), verify  );
+            }
+
+            // Special verifier for point property
+            Consumer verifyPoint = actualValue ->
+            {
+                // The y-coordinate should match the node number
```

Comment:
Is the test simply set up like this or is there another reason why Y is equal to the node number?

Reply:
It was just set up like that to get some controlled variation of the lines